### DISCUSSION TITLE
Depricated Factory::create()

### DISCUSSION THREAD

**Discussion by favayed** [spatie/laravel-short-schedule#39] [Q&A]
Hi, 
I use "laravel/framework": "^8.40"
I after copying the ShortScheduleRunCommand.php file in app/console namespace, received an depricated message from IDE for `Factory::create(); `
So I based on the guide change it with: 
`Loop::get();`

Do you confirm this? And Isn't something threatening my program?

**Comment by ilbeygi** [comment]
I have same problem.
Do you solved this?
